# threshold for the canny edge detector (see https://docs.opencv.org/3.2.0/dd/d1a/group__imgproc__feature.html#ga04723e007ed888ddf11d9ba04e2232de)
canny_threshold: 70

# coordinates of the point, which will be backprojected to the camera image from the specified frame
world_frame_id: "down_rgbd/color_optical"
# world_frame_id: "gps_origin"
world_point:
  x: 0.0
  y: 0.0
  z: 0.0
height_of_formation: 5.0
search_circle_omega:
  # delta angle in radians
  delta: 0.04444444444
  radius: 3.0
search_circle:
  formation_radius: 4.0
  big_circle_radius: 2.0
  maximal_big_circle_radius: 20.0
  minimal_big_circle_radius: 1.0
optimisation:
  steps: 25
rate:
  # how often to publish my topic?
  publish: 5 # [Hz]
  # how often check whether msgs are arriving?
  check_subscribers: 1 # [Hz]
